lib/galaxy_test/api/test_workflows.py

```diff
@@ -7034,12 +7034,14 @@ def test_subworkflow_import_order_maintained(self, history_id):
   outer_input_1:
     type: int
     default: 1
+    optional: true
```

Comment:
@mvdbeek Is this being required now a regression?

Reply:
https://github.com/galaxyproject/galaxy/pull/19092/commits/3add680ef147ba12fc88a9c2fc9a11857baffc5b ?

Reply:
I guess we might have to introduce something to track the meaning of default but no explicit optional ? Like the profile version we have for tools ?

Reply:
Feel free to revert https://github.com/galaxyproject/galaxy/pull/19092/commits/545d783683f8e28b5e6318f04da53abbe9f227ec and I open a separate PR to work on this ?

Reply:
If we never exposed it in the UI and it was only available by hacking ga files for most - I'm fine with just forcing a new behavior.

Reply:
I'll take you up on that but let me know if you want help with that piece. I've pulled out various related things - my test fixes and those two commits and pushed the result back into this branch. The pre-rebase version of my version of this branch is backed up at https://github.com/jmchilton/galaxy/tree/backup_min_max_regex_before_strip_default_handling.

Reply:
Ah, that's a good point, this is only ambiguous in the gxformat2 language where a default value would make the parameter optional previously. So you're okay with having to specify `optional: true` in gxformat2 to create optional inputs ?

Reply:
Yeah - I think so 🤔.